### ISSUE TITLE
Could you please share the coding of combined classification & segmentation model training process?

### ISSUE THREAD

**Issue by solayman-cs** [vbookshelf/Intracranial-Hemorrhage-Analyzer#2]

**Comment by vbookshelf** [comment]
Hi @solayman-cs . Please refer to this notebook. It has all the code that I used to train the model: https://www.kaggle.com/code/vbookshelf/intracranial-hemorrhage-analyzer-tfjs-web-app/notebook
I did this work about 3 years ago so I can't remember exactly what I did. But I didn't do any classification, only segmentation.

**Comment by solayman-cs** [comment]
Hi @vbookshelf, Thanks for your kind response. Yes I understood you had completed it a long ago. In the last part of your mentioned kaggle notebook, you tried to build a combined segmentation & binary classification model. You also built the combined model, but didn't train it. Could you please give the any suggestion for training that combined model? Thanks in advance.

**Comment by vbookshelf** [comment]
These are two resources that show how to train a multi-output Keras model. They explain the training workflow, how to specify different loss functions for different tasks, and how to combine those losses into one value.

1. Book: Deep Learning with Python by Francois Chollet, Chapter 7, Para 7.1.3
2. Medium article: 
https://towardsdatascience.com/multi-output-model-with-tensorflow-keras-functional-api-875dd89aa7c6
